### ISSUE TITLE
Pixel 3 Not recognized as Android Device after upgrade to 1.17.4

### ISSUE THREAD

**Issue by barbarasamson** [flutter/flutter#60306]
My Pixel 3 phone has USB Debugging enabled and has been successfully running Flutter apps loaded from Android Studio. After upgrading to Flutter 1.17.4 the phone was no longer recognized in Android Studio for a Flutter application. The phone was recognized for a simple Android application, but not for Flutter. I downgraded to 1.17.3 and it is recognized in Android Studio again. This is on a Macbook Pro running OS X 10.13.6. In a terminal window, running "flutter devices" the phone does show up as it should:
$ flutter devices
1 connected device:

Pixel 3 • 8AEX0RK47 • android-arm64 • Android 10 (API 29)

But in Android Studio (after upgrade to 1.17.4) there are no connected devices. The Troubleshooter in Android Studio shows the Pixel 3 but says that it does not have USB debugging enabled.

**Comment by iapicca** [comment]
Hi @barbarasamson 
Are you experiencing the issue only with Android Studio / IntelliJ or with VS Code as well?
Thank you

**Comment by barbarasamson** [comment]
I don't know about VS. I don't use it.

**Comment by iapicca** [comment]
Hi @barbarasamson 
could you please run from your terminal
```console
flutter upgrade -f
flutter create test
cd test
flutter run -d 8AEX0RK47 -v
```
making sure that said device is connected,
thank you

**Comment by yellhtutaung** [comment]
> Hi @barbarasamson
> could you please run from your terminal (not from Android Studio)
> 
> ```
> flutter upgrade -f
> flutter create test
> cd test
> flutter run -d 8AEX0RK47 -v
> ```
> 
> making sure that said device is connected,
> thank you

How can i run, command already say flutter is not recognized.

**Comment by iapicca** [comment]
Hi @yellhtutaung 
please add flutter to your path
[windows](https://flutter.dev/docs/get-started/install/windows#update-your-path)
[mac](https://flutter.dev/docs/get-started/install/macos#update-your-path)
[linux](https://flutter.dev/docs/get-started/install/linux#update-your-path)
thank you

**Comment by barbarasamson** [comment]
Hey, this is in response to the request that I upgrade flutter and run a test. I would like to do it, but here's my problem: Flutter has released version 1.17.5 and will only downgrade 1 version. And 1.17.4 is the version that does not recognize my Pixel 3 from my Macbook Pro running OS X 10.13.6. So if the test doesn't work then I will be unable to downgrade to a version of Flutter that works for this computer. It works fine right now on version 1.17.3. This computer does not have the resources to run virtual devices, so I need the connected device to be recognized.

As a side note, I also sometimes have access to a new Macbook Pro running OS X 10.15.5 with Flutter version 1.17.5 and the Pixel 3 is recognized by that one no problem.

**Comment by iapicca** [comment]
HI @barbarasamson 
the reason for this [procedure](https://github.com/flutter/flutter/issues/60306#issuecomment-650948924) is to make sure 
that the issue is indeed within flutter and not just with the intelliJ plugin.

Regardless, given your last message  
I understand that you are not experiencing the issue 
with updated OSX and undated stable version;

I feel safe to close this issue,
if you disagree please write in the comments
possibly providing the info required
and I will reopen it. 
Thank you
